doc/tesseract.1.asc

```diff
@@ -268,7 +277,7 @@ The engine was developed at Hewlett Packard Laboratories Bristol and at
 Hewlett Packard Co, Greeley Colorado between 1985 and 1994, with some more
 changes made in 1996 to port to Windows, and some C\+\+izing in 1998. A
 lot of the code was written in C, and then some more was written in C\+\+.
-The C\+\+ code makes heavy use of a list system using macros. This predates
+The C++ code makes heavy use of a list system using macros. This predates
```

Comment:
I'm no asciidoc expert but the first appears to act as some kind of template. If you change the previous one as well:
```
@@ -1121,9 +1121,9 @@ on read_pattern_list().</p></div>
 <div class="sectionbody">
 <div class="paragraph"><p>The engine was developed at Hewlett Packard Laboratories Bristol and at
 Hewlett Packard Co, Greeley Colorado between 1985 and 1994, with some more
-changes made in 1996 to port to Windows, and some C++izing in 1998. A
-lot of the code was written in C, and then some more was written in C++.
-The C++ code makes heavy use of a list system using macros. This predates
+changes made in 1996 to port to Windows, and some C\+\+izing in 1998. A
+lot of the code was written in C, and then some more was written in C<code>.
+The C</code> code makes heavy use of a list system using macros. This predates
```